src/main/java/tutorlink/command/ListComponentCommand.java

```diff
@@ -0,0 +1,27 @@
+package tutorlink.command;
+
+import java.util.HashMap;
+
+import tutorlink.appstate.AppState;
+import tutorlink.exceptions.TutorLinkException;
+import tutorlink.result.CommandResult;
+
+public class ListComponentCommand extends Command {
+
+    public static final String COMMAND_WORD = "list_component";
+    private static final String SUCCESS_MESSAGE = "Here are your components:";
+    private static final String EMPTY_MESSAGE = "No components found.";
+
+    @Override
+    public CommandResult execute(AppState appState, HashMap<String, String> parameters) throws TutorLinkException {
+        if (appState.components.size() == 0) {
+            return new CommandResult(EMPTY_MESSAGE);
+        }
+        return new CommandResult(SUCCESS_MESSAGE + "\n" + appState.components.toString());
+    }
+
+    @Override
```

Comment:
In ListStudentCommand getArgumentPrefixes() returns null. Should we discuss and standardize this aspect as well?